### PR TITLE
Widget readme addition

### PR DESCRIPTION
**Pull request by ruvenzx**
## Related Issues

Widget readme addition

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,6 +222,136 @@ Descope SDK is automatically refreshes the session token when it is about to exp
...
+
+Note:
+
+- Custom fields also appear in the table.
+
+###### Usage
+
+```vue
+<template>
+	<div class="manage-users-wrapper">
```
Comment (dorsha) [inline README.md:254]: Why the div and h1 are needed? same comment for below

File: README.md [hunk 2]
```diff
@@ -222,6 +222,136 @@ Descope SDK is automatically refreshes the session token when it is about to exp
...
+	</div>
+</template>
+
+<script setup>
+import { UserManagement } from '@descope/vue-sdk';
+
+const tenant = 'tenantId';
+</script>
+
+<style>
```
Comment (dorsha) [inline README.md:266]: Don't think it is needed

**Review by dorsha — COMMENTED** [review]
I think snippets should be changed a bit. @asafshen please review as well.

File: README.md (hunk 2 above)
Comment (ruvenzx) [reply README.md:266]: its in the example like this

File: README.md (hunk 1 above)
Comment (ruvenzx) [reply README.md:254]: its in the example like this

File: README.md (hunk 2 above)
Comment (dorsha) [reply README.md:266]: still, it should not be like that in readme.

File: README.md (hunk 1 above)
Comment (dorsha) [reply README.md:254]: still, it should not be like that in readme.

File: README.md [hunk 3]
```diff
@@ -222,6 +222,136 @@ Descope SDK is automatically refreshes the session token when it is about to exp
...
+
+Widgets are components that allow you to expose management features for tenant-based implementation. In certain scenarios, your customers may require the capability to perform managerial actions independently, alleviating the necessity to contact you. Widgets serve as a feature enabling you to delegate these capabilities to your customers in a modular manner.
+
+Important Note:
+
+- For the user to be able to use the widget, they need to be assigned the `Tenant Admin` Role.
+
+#### User Management
+
+The `User Management` widget will let you embed a user table in your site to view and take action.
```
Comment (asafshen) [inline README.md:235]: lets use `UserManagement`

same to bellow

File: README.md (hunk 1 above)
Comment (asafshen) [reply README.md:254]: I agree with dorsha, we should also remove the header

put only the `<UserManagement tenant="tenant" />`

also no need to bind `:tenant`, let descoper decide how they want to pass it

**Review by asafshen — COMMENTED** [review]
looks good

File: README.md [hunk 4]
```diff
@@ -222,6 +222,109 @@ Descope SDK is automatically refreshes the session token when it is about to exp
...
+
+Widgets are components that allow you to expose management features for tenant-based implementation. In certain scenarios, your customers may require the capability to perform managerial actions independently, alleviating the necessity to contact you. Widgets serve as a feature enabling you to delegate these capabilities to your customers in a modular manner.
+
+Important Note:
+
+- For the user to be able to use the widget, they need to be assigned the `Tenant Admin` Role.
+
+#### User Management
+
+The `UserManagement` widget will let you embed a user table in your site to view and take action.
```
Comment (dorsha) [inline README.md:235]: In react-sdk you use with space, so keep the same, make sure to do the same for all sdks.

image

File: README.md [hunk 5]
```diff
@@ -222,6 +222,109 @@ Descope SDK is automatically refreshes the session token when it is about to exp
...
+###### Usage
+
+```vue
+import { RoleManagement } from '@descope/react-sdk'; ...
+<template>
+	<RoleManagement tenant="tenant-id" />
+</template>
+
+<script setup>
+import { RoleManagement } from '@descope/vue-sdk';
```
Comment (dorsha) [inline README.md:289]: no need for tab?

File: README.md [hunk 6]
```diff
@@ -222,6 +222,109 @@ Descope SDK is automatically refreshes the session token when it is about to exp
...
+
+Note:
+
+- The `Editable` field is determined by the user's access to the role - meaning that project-level roles are not editable by tenant level users.
+- You need to pre-define the permissions that the user can use, which are not editable in the widget.
+
+###### Usage
+
+```vue
+import { RoleManagement } from '@descope/react-sdk'; ...
```
Comment (dorsha) [inline README.md:283]: Here you have import and also below, why?
Need to do the same in all examples, according to best practice.

File: README.md [hunk 7]
```diff
@@ -222,6 +222,109 @@ Descope SDK is automatically refreshes the session token when it is about to exp
...
+The widget lets you:
+
+- Create a new access key
+- Activate / deactivate an existing access key
+- Delete an exising access key
+
+###### Usage
+
+```vue
+import { AccessKeyManagement } from '@descope/react-sdk'; ...
```
Comment (dorsha) [inline README.md:309]: same

File: README.md [hunk 8]
```diff
@@ -222,6 +222,109 @@ Descope SDK is automatically refreshes the session token when it is about to exp
...
+<template>
+	<AccessKeyManagement tenant="tenant-id" />
+</template>
+
+<script setup>
+import { AccessKeyManagement } from '@descope/vue-sdk';
+</script>
+
+<style>
```
Comment (dorsha) [inline README.md:318]: No need for style

File: README.md [hunk 9]
```diff
@@ -222,6 +222,109 @@ Descope SDK is automatically refreshes the session token when it is about to exp
...
+
+- The `Editable` field is determined by the user's access to the role - meaning that project-level roles are not editable by tenant level users.
+- You need to pre-define the permissions that the user can use, which are not editable in the widget.
+
+###### Usage
+
+```vue
+import { RoleManagement } from '@descope/react-sdk'; ...
+<template>
+	<RoleManagement tenant="tenant-id" />
```
Comment (dorsha) [inline README.md:285]: @asafshen why here we don't have widgetId and in react-sdk we do?

Comment (asafshen) [reply README.md:285]: we should have widget 

@nirgur this need to be fixed in the code, right?

Comment (asafshen) [reply README.md:285]: I see that in vue we hard coded, I'll fix
image

Comment (asafshen) [reply README.md:284]: https://github.com/descope/vue-sdk/pull/60

Comment (dorsha) [reply README.md:284]: @ruvenzx please fix according

Comment (ruvenzx) [reply README.md]: @dorsha Done

**Review by dorsha — APPROVED** [review]

**Pull request merged**